src/main/java/org/sagebionetworks/web/client/widget/ReactComponentDiv.java

```diff
@@ -39,6 +40,13 @@ protected void onUnload() {
 		super.onUnload();
 	}
 
+	@Override
+	public void clear() {
```

Comment:
This will fix it. In the relevant component, we call `clear` expecting it to unmount the component tree, but that never happens before this change.